uniflow/op/model/model_server.py

```diff
@@ -474,3 +476,201 @@ def __call__(self, data: List[str]) -> List[str]:
                         out = re.sub(r"\n{3,}", "\n\n", out).strip()
             outs.append(out)
         return outs
+
+class BedrockModelServer(AbsModelServer):
+    """Bedrock Model Server Class.
+    
+    The AWS client authenticates by automatically loading credentials as per the methods outlined here:
+    https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html
+
+    If you wish to use a specific credential profile, please provide the profile name from your ~/.aws/credentials file.
+
+    Make sure that the credentials or roles in use have the necessary policies for Bedrock service access.
+
+    Additionally, it is important to verify that your boto3 version supports the Bedrock runtime.
+    """
+
+    def __init__(self, model_config: Dict[str, Any]) -> None:
+
+        try:
+            # import in class level to avoid installing boto3
+            import boto3
+
+            super().__init__(model_config)
+            self._model_config = BedrockModelConfig(**self._model_config)
+
+            aws_profile = self._model_config.aws_profile
+            aws_region = self._model_config.aws_region if self._model_config.aws_region else None
+            session = boto3.Session(profile_name=aws_profile)
```

Comment:
nit: is it possible to make this configurable for user to use either 

- `profile_name` as implemented
- `aws_access_key_id`, `aws_secret_access_key`, and `aws_session_token` as argument.
- Environment variables `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN`

based on their habit to initialize boto session.

Reply:
It is technically feasible to implement the initialization of a boto3 session. However, it may not align with best practices in coding and is not recommended by boto3. 

Ref: [Link](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#:~:text=%3DSESSION_TOKEN%0A)-,Warning,-ACCESS_KEY%2C%20SECRET_KEY)
ACCESS_KEY, SECRET_KEY, and SESSION_TOKEN are variables that contain your access key, secret key, and optional session token. Note that the examples above do not have hard coded credentials. We do not recommend hard coding credentials in your source code.

Reply:
got it. Then, let's keep your best practice.